src/library/entities/Library.java

```diff
@@ -17,65 +17,67 @@
+    private long nextPatronId;
+    private long nextLoanId;
+    private Date currentDate;
+    
+    private Map<Long, Item> catalog;
+    private Map<Long, Patron> patrons;
+    private Map<Long, Loan> loans;
+    private Map<Long, Loan> currentLoans;
+    private Map<Long, Item> damagedItems;
     
 
     private Library() {
-        CaTaLoG = new HashMap<>();
-        PaTrOnS = new HashMap<>();
-        LoAnS = new HashMap<>();
-        CuRrEnT_LoAnS = new HashMap<>();
-        DaMaGeD_ItEmS = new HashMap<>();
-        NeXt_ItEm_Id = 1;
-        NeXt_PaTrOn_Id = 1;        
-        NeXt_lOaN_Id = 1;        
+        catalog = new HashMap<>();
+        patrons = new HashMap<>();
+        loans = new HashMap<>();
+        currentLoans = new HashMap<>();
+        damagedItems = new HashMap<>();
+        nextItemId = 1;
+        nextPatronId = 1;
+        nextLoanId = 1;
     }
 
     
-    public static synchronized Library GeTiNsTaNcE() {        
-        if (SeLf == null) {
-            Path PATH = Paths.get(lIbRaRyFiLe);            
+    public static synchronized Library getInstance() {
+        if (self == null) {
+            Path PATH = Paths.get(LIBRARY_FILE);
             if (Files.exists(PATH)) {    
-                try (ObjectInputStream LiBrArY_FiLe = new ObjectInputStream(new FileInputStream(lIbRaRyFiLe));) {
+                try (ObjectInputStream libraryFile = new ObjectInputStream(new FileInputStream(LIBRARY_FILE));) {
                 
-                    SeLf = (Library) LiBrArY_FiLe.readObject();
-                    Calendar.GeTiNsTaNcE().sEtDaTe(SeLf.CuRrEnT_DaTe);
-                    LiBrArY_FiLe.close();
+                    self = (Library) libraryFile.readObject();
+                    Calendar.GeTiNsTaNcE().sEtDaTe(self.currentDate);
+                    libraryFile.close();
                 }
                 catch (Exception e) {
                     throw new RuntimeException(e);
                 }
             }
-            else SeLf = new Library();
+            else {
+                self = new Library();
+            }
         }
-        return SeLf;
+        return self;
     }
 
     
-    public static synchronized void SaVe() {
-        if (SeLf != null) {
-            SeLf.CuRrEnT_DaTe = Calendar.GeTiNsTaNcE().GeTdAtE();
-            try (ObjectOutputStream LiBrArY_fIlE = new ObjectOutputStream(new FileOutputStream(lIbRaRyFiLe));) {
-                LiBrArY_fIlE.writeObject(SeLf);
-                LiBrArY_fIlE.flush();
-                LiBrArY_fIlE.close();    
+    public static synchronized void save() {
+        if (self != null) {
+            self.currentDate = Calendar.GeTiNsTaNcE().GeTdAtE();
+            try (ObjectOutputStream libraryFile = new ObjectOutputStream(new FileOutputStream(LIBRARY_FILE));) {
+                libraryFile.writeObject(self);
+                libraryFile.flush();
+                libraryFile.close();
             }
             catch (Exception e) {

```

Comment:
See https://github.com/elliotmitchum/ITC205/pull/35/files#r934038352

Reply:
That is not included in CSG8. We agreed on using option number 1 @elliotmitchum